### PR TITLE
GTW-9660: Add authentication_status_reason field to 3ds object

### PR DESCRIPTION
**Pull request by katy-hughes-cko**
# Description of changes in PR

[//]: # 'Please add your description here'

## Checklist

- [x] Added a changelog entry to the `changelog.md` file in either `abc_spec` or `nas_spec`
- [ ] Contacted the Tech Docs team about any corresponding guides that need to be updated for www.checkout.com/docs

**Contributing options**: Look at [our documentation](https://checkout.atlassian.net/wiki/spaces/PD/pages/2169506663/API+ref+publication+process) for options to contribute to the API reference.

### CODE REVIEW

File: nas_spec/components/schemas/Payments/3dsData.yaml [hunk 1]
```diff
@@ -25,6 +25,10 @@ properties:
         * `A` - Authentication was processed by a stand-in service, and is classed as successful
         * `U` - Unable to perform authentication
     example: Y
+  authentication_status_reason:
+    type: string
+    description: > 
+      Only available as a result of a 3DS2 authentication. The response from the DS or ACS which provides information on why the `authentication_response` field has the specified value. Only available when `authentication_response` is not Y. <a href="https://www.checkout.com/docs/risk-management/3d-secure/sessions/understand-authentication-failures#Common_authentication_failures" target="_blank">See this page</a> for possible values
```
Comment (jason-dantzer-cko) [inline nas_spec/components/schemas/Payments/3dsData.yaml:31]: Can we add a line return after the sentence "...is not Y."?

**Review by jason-dantzer-cko — CHANGES_REQUESTED** [review]

**Review by jason-dantzer-cko — APPROVED** [review]

File: nas_spec/components/schemas/Payments/3dsData.yaml [hunk 2]
```diff
@@ -25,6 +25,11 @@ properties:
         * `A` - Authentication was processed by a stand-in service, and is classed as successful
         * `U` - Unable to perform authentication
     example: Y
+  authentication_status_reason:
+    type: string
+    description: >
```
Comment (sebastian-garcia-cardona-cko) [inline nas_spec/components/schemas/Payments/3dsData.yaml:30]:
```suggestion
    description: |
```

File: nas_spec/components/schemas/Payments/3dsData.yaml [hunk 3]
```diff
@@ -25,6 +25,11 @@ properties:
         * `A` - Authentication was processed by a stand-in service, and is classed as successful
         * `U` - Unable to perform authentication
     example: Y
+  authentication_status_reason:
+    type: string
+    description: > 
+      Only available as a result of a 3DS2 authentication. The response from the DS or ACS which provides information on why the `authentication_response` field has the specified value. Only available when `authentication_response` is not Y.<br/>
```
Comment (sebastian-garcia-cardona-cko) [inline nas_spec/components/schemas/Payments/3dsData.yaml:31]:
```suggestion
      The response from the DS or ACS, which provides information on why you received the given `authentication_response` value. 
      
      This field is only returned as a result of a 3DS2 authentication, and only if the returned `authentication_response` value was not `Y`.
```

File: nas_spec/components/schemas/Payments/3dsData.yaml [hunk 4]
```diff
@@ -25,6 +25,11 @@ properties:
...
+      <a href="https://www.checkout.com/docs/risk-management/3d-secure/sessions/understand-authentication-failures#Common_authentication_failures" target="_blank">See this page</a> for possible values
```
Comment (sebastian-garcia-cardona-cko) [inline nas_spec/components/schemas/Payments/3dsData.yaml:32]:
```suggestion
      Refer to our <a href="https://www.checkout.com/docs/risk-management/3d-secure/sessions/understand-authentication-failures#Common_authentication_failures" target="_blank">authentication failures</a> documentation for more information.
```

File: nas_spec/components/schemas/Payments/3dsData.yaml (hunk 3 above)
Comment (sebastian-garcia-cardona-cko) [reply nas_spec/components/schemas/Payments/3dsData.yaml:31]: Could you also write out what DS stands for here please, followed by the acronym in brackets. Same for ACS too. E.g.

> Strong Customer Authentication (SCA)

**Review by sebastian-garcia-cardona-cko — CHANGES_REQUESTED** [review]

**Review by jason-dantzer-cko — APPROVED** [review]

**Review by sebastian-garcia-cardona-cko — APPROVED** [review]

**Pull request merged**